values/_default.yml

```diff
@@ -11,6 +11,8 @@ blobstore:
   region: ""
   resource_directory_key: cc-resources
   secret_access_key_secret_name:
+  #@overlay/match missing_ok=True
```

Comment:
Is this necessary? I thought this file defined the defaults for values when they are missing, so there's no need to have overlays here.

Reply:
Oh you're right, good catch. Removing it